packages/browser-sdk/src/lib/react/useRoomConnection/index.ts

```diff
@@ -90,9 +91,13 @@ export function useRoomConnection(
                 externalId: roomConnectionOptions.externalId || null,
             }),
         );
+
+        // TODO: remove this in SDK v3. Require developers to call joinRoom() API explicitly instead.
+        store.dispatch(doAppStart());
+
```

Comment:
Should we require developers to call joinRoom, or make it possible to pass in a config option `autoJoin: false` maybe? So that we make it kind of backwards compatible?

(Sorry this is not really relevant for this PR, just thought about it when I saw this comment)

Reply:
I guess these breaking changes remain to be discussed for the version 3 major release 🤞